rdrf/rdrf/forms/widgets/settings_widgets.py

```diff
@@ -1,242 +1,151 @@
 import json
 
 from django.forms import Widget
+from django.forms.renderers import get_default_renderer
 from django.utils.safestring import mark_safe
 from django.utils.translation import gettext as _
 
 from .widgets import TimeWidget
 
 
-class SliderWidgetSettings(Widget):
+class JSONWidget(Widget):
```

Comment:
Can we rename this to `JSONWidgetSettings` as it isn't a widget?

Reply:
Updated.